src/contracts/core/StrategyManager.sol

```diff
@@ -941,8 +941,8 @@ contract StrategyManager is
         }   else {
                 uint256 shareAmount = uint256(sharesDelta);
                 //if change in shares is positive, add the shares
+                //note: _addShares calls increaseDelegatedShares while _removeShares does not.
```

Comment:
Doesn't this mean queued withdrawals won't decrease delegated shares then?

Reply:
can u add a space between `//`

Reply:
I think queuedWithdrawals makes a separate call to decreaseDelecatedSares